downstream/modules/platform/proc-operator-deploy-redis.adoc

```diff
@@ -0,0 +1,25 @@
+= Deploying clustered Redis on {OperatorPlatform}
+
+When you create an {PlatformNameShort} instance through the {OperatorPlatform}, standalone Redis is assigned by default. 
+To deploy clustered Redis, see the following procedure.
+
+//Add a link to the section when ready
+For more information about Redis, refer to Caching and queueing system in the _Planning_ _your_ _installation_ _guide_.
```

Comment:
, see <link>[Caching and queuing system] in the _Planning your installation guide_